baseplate/clients/thrift.py

```diff
@@ -216,20 +252,27 @@ def _call_thrift_method(self: Any, *args: Any, **kwargs: Any) -> Any:
                     result = method(*args, **kwargs)
                 except TTransportException as exc:
                     # the connection failed for some reason, retry if able
+                    success = "false"
```

Comment:
:no_bell: I don't see any similar handling for HTTP clients. Does this work properly if TCP connections drop unexpectedly or DNS can't resolve?